.github/workflows/basic-pr-checks.yml

```diff
@@ -1,4 +1,4 @@
-name: "Basic PR checks"
+name: "pull-request"
```

Comment:
could you move this change to a separate PR since it's unrelated?

Reply:
done